lib/bolt.rb

```diff
@@ -1,5 +1,18 @@
 # frozen_string_literal: true
 
+require 'logging'
+
+# re-open Ruby logging gem
+module Logging
+  # the logging gem always sets itself up to initialize little-plugger
+  # https://github.com/TwP/logging/commit/5aeeffaaa9fe483c2258a23d3b9e92adfafb3b2e
+  class << self
+    # little-plugger calls Gem.find_files, incurring an expensive gem scan
+    # monkey patch Logging to remove the extended method
+    def initialize_plugins; end
```

Comment:
This monkey patch is the ugliest part of this PR, but given this disables arbitrary code loading on the client, for plugins that we don't use, I'm OK with overriding the base method in this case.

Reply:
`LittlePlugger` also allows configuration, but only to the default search path - see https://github.com/TwP/little-plugger/blob/master/lib/little-plugger.rb#L302... that config is hardcoded by the `logging` gem with no ability for a user override https://github.com/TwP/logging/commit/5aeeffaaa9fe483c2258a23d3b9e92adfafb3b2e#diff-35a0271f245bc1e1e062519a373c2512R22

(In testing, even if I manually established a path, I couldn't prevent a heavy scan)